pkg/cmd/cli/backstage/constants.go

```diff
@@ -0,0 +1,12 @@
+package backstage
+
+const (
+	COMPONENT_TYPE     = "model-server"
+	RESOURCE_TYPE      = "ai-model"
+	API_TYPE           = "openapi"
```

Comment:
So should we bother picking any value for an API's `spec.type` ?  

At least with the current examples on the dev cluster, the `definition` fields start with `openapi` since those AI API happen to be expressed as `openapi`.  So in theory we could in can be inferred from that.

Do you know if the same is true for `asyncapi` or `graphql` or `grpc` @johnmcollier ?

thanks

Reply:
I’d assume so.

I’m guessing that we don’t need to store an API type for the CLI, and we just infer about their membership in the model catalog based on what depends on them (e.g. model servers or models)

Reply:
ok ... it makes the filtering on "APIs that are only related to AI" a bit more interesting 

I'll keep it unset for now, and may just open an additional fit and finish item to sort out filtering for API

Reply:
Yeah, I think that's fine.

Reply:
OK I found examples of each of them at https://github.com/backstage/backstage/tree/master/plugins/api-docs/dev ... btw it appears to be both `trpc`  and `grpc`

I think then we can just look at the start of the definition string and set this accordingly.  Otherwise we'll leave blank.

And I'll open a new fit and finish search story for fetching apis from ai components and resources.

Part of next push